joblib/__init__.py

```diff
@@ -58,7 +58,10 @@
    inputs and  outputs: Python functions. Joblib can save their
    computation to disk and rerun it only if necessary::
 
+
       >>> from joblib import Memory
+      >>> import shutil ; import os
+      >>> if os.path.exists('/tmp/joblib'): shutil.rmtree('/tmp/joblib/')
```

Comment:
Sorry I did not mean to change that. I had a problem with the doctests that were not passing so I was trying to debug. I'll look at my diff in more details.